.github/workflows/performance.yml

```diff
@@ -0,0 +1,56 @@
+name: Performance Test
+
+on:
+  pull_request:
+    branches: [master, 'release/**']
+    paths-ignore:
+      - 'docs/**'
+      - '**/*.md'
+
+jobs:
+  performance:
+    if: github.event_name == 'pull_request' || github.event.label.name == 'performance'
```

Comment:
Should we only rely on the label? The pull request can be triggered by any user.

Reply:
We can apply a label if needed. Note that people can run anything based on their PR.

Reply:
> We can apply a label if needed

Now this PR is satisfied, add `performance` and it will run. To run it again you need to remove `performance` and add it again.

> Note that people can run anything based on their PR.

Yes, the idealization is so. But this requires continued enhancements to the performance testing framework.
Maybe we can change it the next time we need to run performance tests for other PRs.